src/index.ts

```diff
@@ -11,61 +11,15 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
 
-import * as v2 from './v2';
-
-/**
- * The `@google-cloud/translate` package has the following named exports:
- *
- * - `{@link TranslationServiceClient}` class - constructor for v3 of the Translation API.
```

Comment:
Can we exclude one file in our copy logic somehow? This one looks to be handwritten artisanal code, I believe we will find this to be the case for several ML libraries -- they're some of the worst offenders.